app/models/hackney/repair_request.rb

```diff
@@ -1,8 +1,13 @@
 class Hackney::RepairRequest
   include ActiveModel::Model
 
-  attr_accessor :reference, :description, :contact, :priority, :work_orders,
-    :property_reference, :created_by_email
+  attr_accessor :reference
```

Comment:
why is it better to do it this way?

Reply:
because if we change or add or remove an property, it doesn't touch the whole line. And we can easily comment above or to the right of it.